src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -2806,4 +2806,25 @@ public boolean getNeverPurgeTombstones()
     {
         return neverPurgeTombstones;
     }
-}
\ No newline at end of file
+
+    /**
+     * Check SSTables whether or not they are misplaced.
+     * @return true if any of the SSTables is misplaced.
+     *         If all SSTables are correctly placed or the partitioner does not support splitting, it returns false.
+     */
+    @Override
+    public boolean isSSTableMisplaced()
```

Comment:
maybe make this `anySSTableMisplaced()` ?

Reply:
I'd prefer to have the method name start with a verb. 
How about `hasMisplacedSSTables()`? It has the same meaning as `isAnySSTableMisplaced()`, but shorter.